spec/api-remote-spec.js

```diff
@@ -494,49 +493,49 @@ describe('remote module', () => {
     let derived = cl.derived
 
     it('can get methods', () => {
-      assert.strictEqual(base.method(), 'method')
+      expect(base.method()).to.equal('method')
     })
 
     it('can get properties', () => {
-      assert.strictEqual(base.readonly, 'readonly')
+      expect(base.readonly).to.equal('readonly')
     })
 
     it('can change properties', () => {
-      assert.strictEqual(base.value, 'old')
+      expect(base.value).to.equal('old')
       base.value = 'new'
-      assert.strictEqual(base.value, 'new')
+      expect(base.value).to.equal('new')
       base.value = 'old'
     })
 
     it('has unenumerable methods', () => {
-      assert(!base.hasOwnProperty('method'))
-      assert(Object.getPrototypeOf(base).hasOwnProperty('method'))
+      expect(base.hasOwnProperty('method')).to.be.false()
+      expect(Object.getPrototypeOf(base).hasOwnProperty('method')).to.be.true()
     })
 
     it('keeps prototype chain in derived class', () => {
-      assert.strictEqual(derived.method(), 'method')
-      assert.strictEqual(derived.readonly, 'readonly')
-      assert(!derived.hasOwnProperty('method'))
+      expect(derived.method()).to.equal('method')
+      expect(derived.readonly).to.equal('readonly')
+      expect(derived.hasOwnProperty('method')).to.be.false()
```

Comment:
tests of this format can be `expect(derived).to.not.have.a.property('method')`

Reply:
for some reason it does not work, I am getting
AssertionError: expected {} to not have property 'method'

Reply:
has to be `expect(derived).to.not.have.own.property('method')`